templates/forms/default/form.html.twig

```diff
@@ -20,7 +20,9 @@
     {% endif %}
 {% endfor %}
 
-{% set action = form.action ?: page.route ~ uri.params %}
+{% if action is null %}
+    {% set action = form.action ?: page.route ~ uri.params %}
+{% endif  %}
```

Comment:
Can be written:

```twig
{% set action = action ?? (form.action ?: page.route ~ uri.params) %}
```

Reply:
This would make it less readable, don't you think?

Reply:
To me, it's far more readable. In many places, we have multiple lines like this, and instead of using like 30 lines, you could write the same in 10 lines.

Reply:
Done. Even if one-liners in PHP does not make any sense to me. :)